src/components/Feed.jsx

```diff
@@ -0,0 +1,153 @@
+import React, { useRef, useState } from "react";
+import { useEffect } from "react";
+
+import styled from "styled-components";
+
+const Feed = (props) => {
+  const [commentText, setCommentText] = useState("");
+  const [commentList, setCommentList] = useState(props.comments);
+  const [isLoad, setIsLoad] = useState(true);
+  const inputRef = useRef(null);
+
+  useEffect(() => {
+    onLoadHandler();
+  }, []);
+
+  const onLoadHandler = () => {
+    const loadImage = new Image();
+    loadImage.src = props.img;
+    loadImage.onload = () => {
+      setIsLoad(false);
+    };
+  };
+
+  const commentInputChangeHandler = () => {
+    setCommentText(inputRef.current.value);
+  };
+
+  const findLastItem = () => {
+    const lastItem = commentList[commentList.length - 1];
+    if (lastItem !== undefined) {
+      return lastItem.id + 1;
+    } else {
+      return 1;
+    }
+  };
+
+  const submitHandler = (event) => {
+    event.preventDefault();
+    setCommentList([
+      ...commentList,
+      { id: findLastItem(), userName: "superUser", comment: commentText },
+    ]);
+    setCommentText("");
```

Comment:
form에 useRef를 사용해서 formRef를 만들고 `formRef.current.reset()`을 사용하는 건 어떨까요?
위처럼 한다면 `commentText`라는 상태를 관리하지 않아도 될 것 같아요

http://tcpschool.com/html-input-types/reset > form의 reset과 동일합니다.
```suggestion
    setCommentList([
      ...commentList,
      { id: findLastItem(), userName: "superUser", comment:  inputRef.current.value },
    ]);
    formRef.current.reset()
```

Reply:
comment input의 value를 상태값을 거치치 않고 바로 사용하는 방식은 실제 돔을 건드리는 방식이라 react의 가상돔을 제어하는 방식이랑 취지가 안맞는것 같습니다.

Reply:
오오 리액트의 방식을 생각하지 못했네요 알려주셔서 감사합니다!